docs/modules/operation/pages/events/sources/snmp-traps.adoc

```diff
@@ -2,23 +2,352 @@
 [[ga-events-sources-snmp-traps]]
 = SNMP Traps
 
-If SNMP-capable devices in the network are configured to send *traps* to {page-component-title}, these traps are transformed into events according to preconfigured rules.
-The `Trapd` service daemon, which lets {page-component-title} receive SNMP traps, is enabled by default.
+{page-component-title} can receive and process SNMP traps/informs from SNMP-capable devices out of the box.
+It receives SNMP traps via the xref:reference:daemons/daemon-config-files/trapd.adoc[Trapd service daemon], which is enabled by default.
 
-IMPORTANT: Disabling the `Trapd` service daemon will render {page-component-title} *incapable* of receiving SNMP traps.
+IMPORTANT: Disabling Trapd renders {page-component-title} incapable of receiving SNMP traps.
 
```

Comment:
This feels fairly redundant.  Of course disabling something would cause it to not function.
```suggestion
```